cmd/init-osm-controller/init-osm-controller_test.go

```diff
@@ -4,49 +4,59 @@ import (
 	"testing"
 
 	tassert "github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"github.com/openservicemesh/osm/pkg/apis/config/v1alpha1"
 )
 
 func TestCreateDefaultMeshConfig(t *testing.T) {
 	assert := tassert.New(t)
 
-	presetMeshConfig := &v1alpha1.MeshConfig{
+	presetMeshConfigMap := &corev1.ConfigMap{
 		TypeMeta: metav1.TypeMeta{
-			Kind:       "MeshConfig",
-			APIVersion: "config.openservicemesh.io/v1alpha1",
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name: presetMeshConfigName,
 		},
-		Spec: v1alpha1.MeshConfigSpec{
-			Sidecar: v1alpha1.SidecarSpec{
-				LogLevel:                      "error",
-				EnvoyImage:                    "envoyproxy/envoy-alpine:v1.18.3",
-				InitContainerImage:            "openservicemesh/init:v0.9.1",
-				EnablePrivilegedInitContainer: false,
-				MaxDataPlaneConnections:       0,
-				ConfigResyncInterval:          "2s",
-			},
-			Traffic: v1alpha1.TrafficSpec{
-				EnableEgress:                      true,
-				UseHTTPSIngress:                   false,
-				EnablePermissiveTrafficPolicyMode: true,
-			},
-			Observability: v1alpha1.ObservabilitySpec{
-				EnableDebugServer: false,
-				Tracing: v1alpha1.TracingSpec{
-					Enable: false,
-				},
-			},
-			Certificate: v1alpha1.CertificateSpec{
-				ServiceCertValidityDuration: "24h",
-			},
+		Data: map[string]string{
+			presetMeshConfigJSONKey: `{
+"sidecar": {
+  "enablePrivilegedInitContainer": false,
+  "logLevel": "error",
+  "maxDataPlaneConnections": 0,
+  "envoyImage": "envoyproxy/envoy-alpine:v1.18.3",
+  "initContainerImage": "openservicemesh/init:v0.9.0",
+  "configResyncInterval": "2s"
+},
+"traffic": {
+	"enableEgress": true,
+	"useHTTPSIngress": false,
+	"enablePermissiveTrafficPolicyMode": true,
+	"outboundPortExclusionList": [],
+	"inboundPortExclusionList": [],
+	"outboundIPRangeExclusionList": []
+  },
+  "observability": {
+	"enableDebugServer": false,
+	"osmLogLevel": "trace",
+	"tracing": {
+   "  enable": false
```

Comment:
Spaces in front of `enable`

Reply:
+1. Could you address this before merge.

Reply:
fixed